dbhydra/dbhydra_core.py

```diff
@@ -1849,14 +1958,16 @@ def split_assign(variable_assign):
             return (variable, value)
 
         df = self.select_to_df()
+        before_delete_count = len(df)
         if where is None:
             df = df.iloc[0:0]
-            print(df)
+            deleted_count = before_delete_count - 0
```

Comment:
??? what

Reply:
I've written it `before_delete_count - 0` to keep it more explicit but obviously achieved the opposite effect ;)